apps/editing-toolkit/editing-toolkit-plugin/global-styles/static/attach-tracks-events-to-fonts-links.js

```diff
@@ -0,0 +1,26 @@
+( function () {
+	const customizerGlobalStylesBlockEditorLinkElem = window.parent.document.getElementById(
```

Comment:
Thank you for taking a second look at this pull request, @kwight.

The reason my earlier attempts at this javascript file didn't work: I didn't originally realize that I would need to interact with the parent iframe to find the DOM element.

But one concern I've left with: This file isn't babel transpiled right now. I'm concerned since most javascript files are being served through the dist folder, and I'm breaking that standard.

But in order to babel transpile this file, I will basically need to create a whole new package to be built. I already got that working before, so I think I could do it quickly. I'm just concerned that maybe it's not a good idea since it might introduce a lot of unexpected complexity somewhere along the line (maybe even after merging)? Or maybe it's not a good idea since this will be deprecated? Just wanted to get your thoughts.

The package definitions are in this [package.json](https://github.com/Automattic/wp-calypso/blob/trunk/apps/editing-toolkit/package.json). We use an npm-run-all script to run through every package in that file. Lines 28 -> 31 are an example of how we achieve multiple packages in one folder.

Reply:
We discussed this, and decided that doing it properly was the way to go, even if it means a new bundle.

Reply:
This should be all set

Reply:
Actually hold off one minute...